terraform/modules/marklogic/dap_export_s3.tf

```diff
@@ -0,0 +1,113 @@
+module "dap_export_bucket" {
+  source = "../s3_bucket"
+
+  bcuket_name            = "dluhc-delta-dap-export-${var.environment}"
+  access_log_bucket_name = "dluhc-delta-dap-export-access-logs-${var.environment}"
+}
+
+resource "aws_ssm_maintenance_window" "dap_s3_upload" {
+  name              = "marklogic-dap-s3-upload-${var.environment}"
+  schedule          = "cron(00 06 ? * * *)" # 6 AM every day
+  schedule_timezone = "Etc/UTC"
+  duration          = 2
+  cutoff            = 1
+}
+
+resource "aws_ssm_maintenance_window_target" "ml_server" {
+  window_id     = aws_ssm_maintenance_window.dap_s3_upload.id
+  name          = "marklogic-dap-s3-upload-${var.environment}"
+  description   = "This should contain one MarkLogic server from the ${var.environment} environment"
+  resource_type = "INSTANCE"
+
+  targets {
+    key    = "tag:Name"
+    values = ["MarkLogic-ASG-1"]
+  }
+
+  targets {
+    key    = "tag:environment"
+    values = [var.environment]
+  }
+}
+
+resource "aws_ssm_maintenance_window_task" "dap_s3_upload" {
+  window_id       = aws_ssm_maintenance_window.dap_s3_upload.id
+  max_concurrency = 1
+  max_errors      = 0
+  priority        = 1
+  task_arn        = "AWS-RunShellScript"
+  task_type       = "RUN_COMMAND"
+  cutoff_behavior = "CONTINUE_TASK"
+
+  targets {
+    key    = "WindowTargetIds"
+    values = [aws_ssm_maintenance_window_target.ml_server.id]
+  }
+
+  task_invocation_parameters {
+    run_command_parameters {
+      comment         = "MarkLogic DAP S3 data upload"
+      timeout_seconds = 60
+
+      service_role_arn = aws_iam_role.dap_sns_publish.arn
+      notification_config {
+        notification_arn    = aws_sns_topic.dap_s3_upload_notifications.arn
+        notification_events = ["TimedOut", "Cancelled", "Failed"]
+        notification_type   = "Command"
+      }
+
+      parameter {
+        name = "commands"
+        # TODO DT-23: Confirm location once ML job is set up
+        values = ["aws s3 cp /export-data \"s3://${module.dap_export_bucket.bucket}/data\" --recursive"]
```

Comment:
Why would it be different from Datamart's setup? I mean, yes, we can set `data-export-path` to something different in dluhc.properties, but do you anticipate any need to?

Reply:
Ah I see in your PR description you mentioned /mnt/backup
But prod.properties has `data-export-path=/delta/export` instead. Which will probs be fine? Don't know if we need the user data to create that folder beforehand. Or if xqy handles creating necessary directories

The ftp script then moves the export files to /mnt/backup after the successful export. How about we persist it in S3? We have one folder in the bucket for DAP to get that day's data and a second folder that DAP doesn't get access to, to keep all exports?